Kronos-Shifts-Connector/README.md

```diff
@@ -106,13 +106,15 @@ The table below outlines the required permissions necessary for the Azure AD app
 
 |Scope|Application/Delegated|Function|
 |-----|---------------------|--------|
-|Group.Read.All|Delegated|Allows application to list groups and read properties and all group memberships on behalf of the signed-in user (tenant admin).|
-|Group.ReadWrite.All|Delegated|Allows the application to create groups and read all group properties and memberships on behalf of the signed-in user (tenant admin).|
-|WorkforceIntegration.Read.All|Delegated|Allows for workforce integrations to be retrieved from Microsoft Graph.|
-|WorkforceIntegration.ReadWrite.All|Delegated|Allows for workforce integrations to be created and registered with Microsoft Graph.|
+|Group.Read.All|Delegated|Allows application to list groups and read properties and all group memberships on behalf of the signed-in user (tenant admin)|
```

Comment:
Group.Read.All is included in Group.ReadWrite.All. You don't need both.

Reply:
Same with WorkforceIntegration and Schedule.

Reply:
@aosolis - ack'd. Thanks for the information and the help!